src/js/components/Menu/Menu.js

```diff
@@ -368,16 +368,22 @@ const Menu = forwardRef((props, ref) => {
         // eslint-disable-next-line react/no-array-index-key
         key={groupIndex}
         {...theme.menu.group?.container}
-        border={
-          groupIndex > 0
-            ? {
-                side: 'top',
-                color: theme.menu.group?.separator?.color,
-                size: theme.menu.group?.separator?.size,
-              }
-            : undefined
-        }
       >
+        <Box
+          margin={{
+            horizontal: 'small',
+            bottom: index === 0 ? 'none' : 'small',
```

Comment:
The container of the group already has vertical padding. It doesn't feel like we need to add this bottom margin also.

Reply:
Fixed in latest commit